receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -0,0 +1,595 @@
+name: dockerstatsreceiver
+
+sem_conv_version: 1.6.1
+
+# Note: there are other, additional resource attributes that the user can configure through the yaml
+resource_attributes:
+  container.runtime:
+    description: "The runtime of the container. For this receiver, it will always be 'docker'."
+    type: string
+  container.id:
+    description: "The ID of the container."
+    type: string
+  container.image.name:
+    description: "The name of the docker image in use by the container."
+    type: string
+  container.name:
+    description: "The name of the container."
+    type: string
+  container.hostname:
+    description: "The hostname of the container."
+    type: string
+
+attributes:
+  core:
+    description: "The CPU core number when utilising per-CPU metrics."
+    type: string
+  device_major:
+    description: "Device major number."
+    type: string
+  device_minor:
+    description: "Device minor number."
+    type: string
+  interface:
+    description: "Network interface."
+    type: string
+
+metrics:
+  # CPU
+  container.cpu.usage.system:
+    enabled: true
+    description: "System CPU usage."
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  container.cpu.usage.total:
+    enabled: true
+    description: "Total CPU time consumed."
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  container.cpu.usage.kernelmode:
+    enabled: true
+    description: >-
+      Time spent by tasks of the cgroup in kernel mode (Linux). 
+      Time spent by all container processes in kernel mode (Windows).
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  container.cpu.usage.usermode:
+    enabled: true
+    description: >-
+      Time spent by tasks of the cgroup in user mode (Linux). 
+      Time spent by all container processes in user mode (Windows).
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+  container.cpu.usage.percpu:
+    enabled: false
+    description: "Per-core CPU usage by the container."
+    unit: ns
+    sum:
+      value_type: int
+      monotonic: true
+      aggregation: cumulative
+    attributes:
+      - core
+  container.cpu.throttling_data.periods:
+    enabled: true
+    description: "Number of periods with throttling active."
+    unit: "1"
```

Comment:
It essentially means no unit. Like for a percentage you would use this unit. It's wrong, and this should be `{periods}`, but that change (and a bunch of others) is for another PR. Right now I am just recreating what is already there with mdatagen. I didn't want to mix up the conversations of "what are the correct metrics" and "how to re-implement", otherwise this PR would be way to huge and complicated.